.github/workflows/memory-tests.yml

```
@@ -0,0 +1,23 @@
+on:
+  pull_request:
+    branches: ['main']
+
+jobs:
+  memory-tests:
+    name: Memory
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)